plugins/design-system/styles/shame.less

```diff
@@ -1,5 +1,4 @@
 // Shame CSS
-// A place to put quick CSS iterations and cleanup later
```

Comment:
Can we delete this file? In this PR https://github.com/dcos/dcos-ui/pull/2388/files you've introduced a separate file for the `do-dont` component. Doesn't matter how messy css is it should all go there.

Reply:
Yes :)